bin/MadGraph5_aMCatNLO/cards/production/13TeV/exo_DisappTrks/C1C1_C1N1_2j_proc_card.dat

```diff
@@ -0,0 +1,43 @@
+#************************************************************
+#*                        MadGraph 5                        *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 1.5.3                 2012-11-01         *
+#*                                                          *
+#*    The MadGraph Development Team - Please visit us at    *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph 5                *
+#*                                                          *
+#*     run as ./bin/mg5  filename                           *
+#*                                                          *
+#************************************************************
+import model mssm
+define p = p b b~
+define j = j b b~
```

Comment:
The rest of the cards look like they are set up for 4 flavour scheme.  The mssm model has massive b-quarks by default, so this would be 4 flavour scheme.  But in that case the proton and jet definition should not include the b quarks.

If you want the proton and jet definition to include the b quarks (also so that the additional partons include also b jets) then you should switch to "import model mssm-no_b_mass" and change also the pdfs and maxjetflavour for consistent 5 flavour scheme.

(5 flavour scheme is the default in CMS for jet matching with SM processes at least)